docs/iot-devkit/devkit-get-started.md

```diff
@@ -32,27 +32,46 @@ To connect the IoT DevKit to your computer:
 
 ![Hardware connections](media/iot-devkit-get-started/connect.jpg)
 
-## Configure Wi-Fi
+## Get Started with Azure IoT Hub
 
-IoT projects rely on internet connectivity. Use AP Mode on the DevKit to configure and connect to Wi-Fi.
+1. Follow this [tutorial](https://docs.microsoft.com/en-us/azure/iot-hub/quickstart-send-telemetry-node#create-an-iot-hub) to
```

Comment:
just copy the [Create an IoT hub] and [Register a device] sections from the docs, jump out is not a good experience. Docs also do the same thing that dup these 2 steps among different language version of send telemetry tutorial